test/jdk/javax/swing/plaf/metal/MetalBorders/ScaledMetalBorderTest.java

```diff
@@ -0,0 +1,351 @@
+    private static JLabel scale;
+
+    public static void main(String[] args) throws AWTException,
+            InterruptedException, InvocationTargetException {
+        try {
+            UIManager.setLookAndFeel("javax.swing.plaf.metal.MetalLookAndFeel");
+            JFrame.setDefaultLookAndFeelDecorated(true);
+            JDialog.setDefaultLookAndFeelDecorated(true);
+        } catch (Exception e) {
+            System.out.println("Metal LAF class not supported");
+            return;
+        }
+
+        try {
+            robot = new Robot();
+            robot.setAutoDelay(100);
+            uiScale = System.getProperty("sun.java2d.uiScale");
+            scale = new JLabel("UI Scale: " + uiScale);
+
+            //Case 1: JFrame
+            SwingUtilities.invokeAndWait(ScaledMetalBorderTest::createFrame);
+            robot.waitForIdle();
+            robot.delay(100);
+            runTests("JFrame");
+
+            if (!errorLog.isEmpty()) {
+                saveScreenCapture("Frame_uiScale_" + uiScale + ".png");
+                System.err.println("JFrame at uiScale: " + uiScale);
+                throw new RuntimeException("Following error(s) occurred: \n"
+                        + errorLog);
+            }
+            errorLog.setLength(0); // to clear the StringBuffer before next test.
+
+            //Case 2: JDialog
+            SwingUtilities.invokeAndWait(ScaledMetalBorderTest::createDialog);
+            robot.waitForIdle();
+            robot.delay(100);
+            runTests("JDialog");
+
+            if (!errorLog.isEmpty()) {
+                saveScreenCapture("Dialog_uiScale_" + uiScale + ".png");
+                System.err.println("JDialog at uiScale: " + uiScale);
+                throw new RuntimeException("Following error(s) occurred: \n"
+                        + errorLog);
+            }
+            errorLog.setLength(0); // to clear the StringBuffer before next test.
+
+            //Case 3: JInternalFrame
+            SwingUtilities.invokeAndWait(ScaledMetalBorderTest::createJInternalFrame);
+            robot.waitForIdle();
+            robot.delay(100);
+            runTests("JIF");
+
+            if (!errorLog.isEmpty()) {
+                saveScreenCapture("JIF_uiScale_" + uiScale + ".png");
+                System.err.println("JInternalFrame at uiScale: " + uiScale);
+                throw new RuntimeException("Following error(s) occurred: \n"
+                        + errorLog);
+            }
+        } finally {
+            SwingUtilities.invokeAndWait(() ->{
+                if (jFrame != null) {
+                    jFrame.dispose();
+                }
+                if (jDialog != null) {
+                    jDialog.dispose();
+                }
+            });
+            robot.delay(200);
+        }
+    }
+
+    private static void runTests(String windowType) throws InterruptedException,
+                                                         InvocationTargetException {
```

Comment:
```suggestion
    private static void runTests(String windowType) throws InterruptedException,
                                                           InvocationTargetException {
```
Should they rather be aligned?